website/docs/foundations/icons/index.md

```diff
@@ -2,6 +2,7 @@
 title: Icons
 layout:
   sidecar: false
+caption: Icons can be used to support and enhance meaning...
 ---
```

Comment:
Is something along these lines what we want? (I came up with this myself but it's not finished so I added the "...")